.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
+
+install_status=1
+
+SRC="${PWD}/_out/build_install"
+
+# Expect the host filesystem to be mounted in following path.
+HOST_MOUNT=/host
+
+# Some files are copied over /usr which on Red Hat CoreOS (rhcos) is mounted
+# read-only by default. So re-mount it as read-write, otherwise files won't
+# get copied.
+mount -o remount,rw $HOST_MOUNT/usr
+
+# The host's '/opt' and '/usr/local' are symlinks to, respectively, '/var/opt'
+# and '/var/usrlocal'. Adjust the installation files accordingly.
+#
+cd $SRC
+if [ -d 'opt' ]; then
+	mkdir -p var
+	mv opt var
+fi
+
+if [ -d "usr/local" ]; then
+	mkdir -p var
+	mv usr/local var/usrlocal
+fi
+
+# Copy the installation over the host filesystem.
+rsync -a . $HOST_MOUNT
+
+# Ensure the binaries are searcheable via PATH. Notice it expects that
+# kata has been built with PREFIX=/opt/kata.
+#
+chroot $HOST_MOUNT sh -c 'for t in /opt/kata/bin/*; do ln -s $t /var/usrlocal/bin/; done'
```

Comment:
Nit: Might be worth quoting `"$t"` to protect against spaces.

Reply:
@jodh-intel I forgot the only 2 needed principles for coding shell script. 1st rule: "You must quote variables"; 2nd rule: "Don't forget the 1st rule" :D